pages/validators/how_to_set_up_full_node.md

```diff
@@ -4,60 +4,124 @@
 1. Linux (Ubuntu Server 22.04.3 recommended)
 2. 8-cpu (ARM or x86_64), 64 GB RAM, 500 GB SSD NVME Storage
 
-## Get the dydxprotocold binary and initialize the data directory
-> **Note:** the example values below align with the **deployment by DYDX token holders**. For alternatives, please visit the [Network Constants page](https://docs.dydx.exchange/network/network_constants).
 
+## Install Full Node
+Install a full node to participate in the network.  The following steps will guide you through the process of setting up a full node. Refer or use the script at https://github.com/dydxprotocol/v4-chain/blob/main/protocol/scripts/create_full_node.sh for a more automated process.
+
+### Step 1: Update System and Install Dependencies
+
+```bash
+sudo apt-get -y update
+sudo apt-get install -y curl jq lz4
+```
+
+### Step 2: Install Go
+Install Go and set the environment variables.  This allows cosmovisor to be installed. amd64 architecture is used in this example.
+```bash
+wget https://golang.org/dl/go1.22.2.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.22.2.linux-amd64.tar.gz
+rm go1.22.2.linux-amd64.tar.gz
+echo 'export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin' >> $HOME/.bashrc
+eval "$(cat $HOME/.bashrc | tail -n +10)"
+```
+
+### Step 3: Install Cosmovisor
+Cosmovisor is a process manager for Cosmos SDK-based blockchains. It allows for the automatic updating of the binary without downtime.  The following command installs Cosmovisor.
+```bash
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0
+```
+
+### Step 4: Download dydxprotocold Binaries and Initialize Node
 1. From https://github.com/dydxprotocol/v4-chain/releases/ | Look for the `protocol` assets.
-2. For example, as of `10/19/2023`, this was the correct binary to use:
+
+2. For example, as of `04/26/2024`, this was the correct binary to use:
```

Comment:
Can we provide instructions on how to figure out which version to use at any given time?